MIP/mip-125/README.md

```diff
@@ -0,0 +1,237 @@
+
+
+### Reward System
+
+The reward system has two main components: how rewards are calculated per validator, and how the reward rate is determined.
+
+#### Validator Reward Calculation
+
+Validator rewards are calculated using the following formula implemented in `aptos-move/framework/aptos-framework/sources/stake.move`:
+
+```
+rewards_amount = (stake_amount * rewards_rate * num_successful_proposals) / (rewards_rate_denominator * num_total_proposals)
+```
+
+This formula is implemented in the `calculate_rewards_amount()` function at lines 1751-1774 in `aptos-move/framework/aptos-framework/sources/stake.move`.
+
+**Parameters** (names do not reflect codebase names):
+
+- `stake_amount`: Validator's active stake
+- `rewards_rate`: Numerator of the reward rate fraction
+- `rewards_rate_denominator`: Denominator of the reward rate fraction  
+- `num_successful_proposals`: Validator's successful block proposals in the epoch
+- `num_total_proposals`: Validator's total block proposals in the epoch
+
+#### Reward Rate Determination
+
+The per-epoch reward rate is automatically calculated using the following formula :
+
+```
+rewards_rate_numerator = (target_apr_percentage * rewards_rate_denominator / 100) / num_epochs_in_a_year
+```
+
+**Parameters** (names do not reflect codebase names):
+
+- `target_apr_percentage`: <VALUE> (for <VALUE>% APR)
+- `rewards_rate_denominator`: 1_000_000_000 (for precision)
+- `num_epochs_in_a_year`: 4_380 (based on 2-hour epochs)
+
+This calculation is implemented in `aptos-move/vm-genesis/src/lib.rs` lines 535-542 and automatically converts the configured APY percentage to the appropriate per-epoch reward rate.
+
+**Example Reward Rate Values**
+
+For a <VALUE>% APR with 2-hour epochs:
+- **rewards_rate**: 22_831
+- **rewards_rate_denominator**: 1_000_000_000
+- **Per-epoch rate**: 0.000022831 (0.0022831%)
+- **Annual rate**: <VALUE>% APR
+
+---
+
+### Updates at Genesis (may not be required)
+
+To implement a <VALUE> APR reward system, the following change MUST be made:
+
+#### Required Configuration Change
+
+Update the genesis configuration file to set the target APR to <VALUE>%:
+
+- **File**: `terraform/helm/genesis/values.yaml`
+- **Line**: 33
+- **Change**: Set `rewards_apy_percentage: <VALUE>`
+- **Purpose**: Configure the target APR for genesis
+
+**Derived Parameters:**
+- **Target APR**: <VALUE>% per year
+- **Epoch Duration**: 2 hours (7_200 seconds) - configured in `epoch_duration_secs: 7200`
+- **Epochs per Year**: ~4_380 epochs (=31_536_000 / 7_200)
+- **Per-Epoch Rate**: <VALUE>% / 4_380 per epoch (calculated automatically)
+
+**Result**: All other components will automatically use the new reward rate without any code changes.
+
+---
+
+### Updates at Runtime
+
+After genesis, there are **two different ways** to update reward rates, depending on which system is active:
+
+#### Path 1: Manual Governance Updates
+
+**When Active**: `periodical_reward_rate_decrease_enabled()` is **FALSE** (current system)
```

Comment:
Note that

Whilst we still have core_resource signer, these proposals will be issued and signed off by the core_resources signer _until_ we have enabled partial OR full governance (partial is recommended initially as it works better for small validator sets). 

We intend to do this _after_ Move2 is out and the Migration is complete on mainnet.